sklearn/metrics/tests/test_score_objects.py

```diff
@@ -241,8 +221,7 @@ def test_check_scoring_and_check_multimetric_scoring():
         estimator = LinearSVC(random_state=0)
         estimator.fit([[1], [2], [3]], [1, 1, 0])
 
-        scorers, is_multi = _check_multimetric_scoring(estimator, scoring)
-        assert is_multi
+        scorers = _check_multimetric_scoring(estimator, scoring)
         assert isinstance(scorers, dict)
         assert sorted(scorers.keys()) == sorted(list(scoring))
         assert all([isinstance(scorer, _PredictScorer)
```

Comment:
The part checking for the error below should probably deserve a separate test I think.